source/testing/testing-components.md

```diff
@@ -258,7 +258,7 @@ is being returned from the service is reflected in the component output.
 ```tests/integration/components/location-indicator-test.js
 test('should reveal current location', function(assert) {
   this.render(hbs`{{location-indicator}}`);
-  assert.equal(this.$().text().trim(), 'You currently are located in New York, USA');
+  assert.equal(this.$().text().trim(), 'Location: New York, USA');
```

Comment:
Yeah, I can see how it would feel arbitrary. Basically, it was just done to reduce line length so the examples didn't scroll. I felt like it conveyed the same message, but was easier to grok if you can see all of the line at once, without scrolling or wrapping.